Orange/widgets/regression/owrandomforestregression.py

```diff
@@ -1,35 +1,24 @@
-from PyQt4 import QtGui
-from Orange.data import Table
-from Orange.regression.random_forest import (RandomForestRegressionLearner,
-                                             RandomForestRegressor)
-from Orange.widgets.classify.owrandomforest import OWRandomForest
+from Orange.regression.random_forest import RandomForestRegressionLearner
+from Orange.widgets.classify import owrandomforest
 from Orange.widgets import settings
 
 
-class OWRandomForestRegression(OWRandomForest):
+class OWRandomForestRegression(owrandomforest.OWRandomForest):
     name = "Random Forest Regression"
     description = "Random forest regression algorithm."
+    icon = "icons/RandomForest.svg"
+    priority = 40
 
     LEARNER = RandomForestRegressionLearner
-    outputs = [("Learner", LEARNER),
-               ("Model", RandomForestRegressor)]
+    OUTPUT_MODEL_NAME = "Model"
```

Comment:
I am not sure that separate classes are the way to go, as I would rather see a single widget that is used for both regression and classification (but this is out of scope for this PR).

Regression widgets extending classification widgets is IMHO good enough, so I would just make sure that all regression widgets use the same name for model output (I vote for Predictor).